torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1043,6 +1053,7 @@ def _parse_five_crop_size(size: List[int]) -> List[int]:
 def five_crop_image_tensor(
     img: torch.Tensor, size: List[int]
 ) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor]:
+    # datumbox: Add TODO to consider breaking BC and return a list to align with ten_crop
```

Comment:
cc @vfdev-5 / @pmeier

Reply:
I'll do it.

Reply:
See https://github.com/pytorch/vision/pull/6492/commits/024f2f450bc2aa69c7a21983432cd05c685ff603

Reply:
Addressed by #6492